prody/proteins/pdbfile.py

```diff
@@ -162,6 +162,10 @@ def parsePDBStream(stream, **kwargs):
     elif model != 0:
         ag = AtomGroup(str(kwargs.get('title', 'Unknown')) + title_suffix)
         n_csets = 0
+    if "needfullname" in kwargs:
```

Comment:
This fix should not be added, it will cause some unexpected error while writing to a PDB file.